providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/grpc/GrpcResolver.java

```diff
@@ -86,53 +94,56 @@ public void shutdown() throws Exception {
      * Boolean evaluation from grpc resolver.
      */
     public ProviderEvaluation<Boolean> booleanEvaluation(String key, Boolean defaultValue,
-            EvaluationContext ctx) {
+                                                         EvaluationContext ctx) {
         ResolveBooleanRequest request = ResolveBooleanRequest.newBuilder().buildPartial();
 
-        return resolve(key, ctx, request, this.connector.getResolver()::resolveBoolean, null);
+
+        return resolve(key, ctx, request, ((ServiceGrpc.ServiceBlockingStub) connector.getResolver())::resolveBoolean,
```

Comment:
[question] can we prevent the casting here?

Reply:
i don't know. Note, that currently the the only reason why the blocking stub stuff is part of the `GrpcConnector` is to make the tests in `FlagdProviderTest` happy. They rely on a mocked stub given back by the `GrpcConnector`. I have an idea how to make the tests green without that. Then we can delete this code as well and let the caller create the stubs. I could do that as a follow up.

Reply:
```
private final GrpcConnector<ServiceGrpc.ServiceStub, ServiceGrpc.ServiceBlockingStub> connector;
```
removes the need to cast ;)